advisories/advisory-side.php

```diff
@@ -1,5 +1,7 @@
 <br/><p>ownCloud server 9.1.3</p>
-<a href="/security/advisory?id=oc-sa-2016-022">Bypass received read-only share permissions using read-write reshare</a><br/>
```

Comment:
@Peter-Prochaska still removed ? Should it be moved to the previous version 9.1.2 maybe ?